test/test_fakeredis.py

```diff
@@ -2112,6 +2112,37 @@ def test_zadd_incr(r, ch):
     assert zadd(r, 'foo', {'three': 1.0}, incr=True, xx=True, ch=ch) == 4.0
 
 
+def test_zpopmin_empty(r):
+    assert r.zpopmin('foo') == []
+
+
+def test_zpopmin_default_count(r):
+    zadd(r, 'foo', {'one_a': 1})
+    zadd(r, 'foo', {'one_b': 1})
+    zadd(r, 'foo', {'two_a': 2})
+    assert r.zpopmin('foo') == [(b'one_a', 1.0)]
+    assert r.zcard('foo') == 2
```

Comment:
Rather test the actual contents (with `r.zrange(0, -1)`) to ensure that the right elements were removed.